plugin.video.embycon/addon.xml

```diff
@@ -1,25 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon  id="plugin.video.embycon" 
         name="EmbyCon"
-        version="1.9.32"
+        version="1.9.33"
         provider-name="Team B">
   <requires>
     <import addon="xbmc.python" version="2.25.0"/>
   </requires>
   <extension point="xbmc.python.pluginsource" library="default.py">
-        <provides>video audio</provides>
-  </extension>
-  <extension point="xbmc.service" library="service.py" start="login">
+        <provides>video</provides>
   </extension>
   <extension point="xbmc.addon.metadata">
+	<broken>Future versions please see http://kodi.emby.media/</broken>
     <platform>all</platform>
     <language>en</language>
     <license>GPL-2.0-only</license>
     <forum>https://github.com/faush01/plugin.video.embycon/issues</forum>
     <website>https://github.com/faush01/plugin.video.embycon/wiki</website>
     <source>https://github.com/faush01/plugin.video.embycon</source>
     <summary lang="en_GB">Browse and play your Emby server media library.</summary>
-    <description lang="en_GB">An addon to allow you to browse and playback your Emby (www.emby.media) Movie, TV Show and Music collections.</description>
+    <description lang="en_GB">Future versions please see http://kodi.emby.media/</description>
```

Comment:
Just a note, broken addons do not show on the addon browser

Reply:
I understand, thanks